core/src/main/scala/quasar/blobstore/AsyncHttpClientBuilder.scala

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2020 Precog Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.blobstore
```

Comment:
Shall we move this into `gcs` lib for now? Then we don't need the 2 http4s libs and `proxy-vole` in `core` anymore.

Reply:
I thought about this, but what's going to happen when we have to write another connector that needs an async http client?

Reply:
Yeah, we would need to tackle that then. Personally I would like it if we started to create microlibs for utils like this. That would even help now, because this code is already duplicated but I'm fine if we postpone that for now.

Reply:
I see what you mean. So ideally you would like to see the async http4s client code in its own separate repo?
I'll move it this file to be under gcs.

Reply:
yeah, when it avoids duplication, i.e. for utils.

As an aside, I'd use 1 repos that would contain multiple modules, one of them for http4s utils like this one.